src/assets/assets.controller.spec.ts

```diff
@@ -125,6 +158,12 @@ describe('AssetsController', () => {
           },
         ])
       )[0];
```

Comment:
Can you add a test for the `find` where you get a failure because block isn't found? Also I think we could add a an error message to that exception saying block wasn't found for where asset was created, might be helpful if we ever have to debug.